src/platform/forms-system/src/js/utilities/date/index.js

```diff
@@ -1,18 +1,18 @@
 import moment from 'moment';
 
 export const months = [
-  { label: 'Jan', value: 1 },
-  { label: 'Feb', value: 2 },
-  { label: 'Mar', value: 3 },
-  { label: 'Apr', value: 4 },
-  { label: 'May', value: 5 },
-  { label: 'Jun', value: 6 },
-  { label: 'Jul', value: 7 },
-  { label: 'Aug', value: 8 },
-  { label: 'Sep', value: 9 },
-  { label: 'Oct', value: 10 },
-  { label: 'Nov', value: 11 },
-  { label: 'Dec', value: 12 },
+  { label: 'Jan', value: 1, text: 'January' },
```

Comment:
These are good improvements. Some of the longest labels are getting cut off when the select is closed. I'll add a screenshot to https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/394 to capture that new success criteria.

Reply:
I agree! Looks like we shouldn't have even been abbreviating April, May, June, and July:

https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years

As a sidenote, do we need the `label` anymore?

Reply:
I don't know if it's used by other apps... so I left it.

Reply:
@Mottie @1Copenut I still see the longer month names being cut off when the select is closed.  Is that planned to be fixed with this PR?
![image](https://user-images.githubusercontent.com/2481110/107556481-76fd7f80-6ba6-11eb-8b9c-ab96bf8bb3b9.png)

Reply:
Making the Month select wider will require a separate PR to the `veteran-facing-services-tools` repo to modify this entry:

```css
.form-datefield-month {
  width: 12rem;
}
```

Reply:
See https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/588